kfs-core/src/main/java/edu/arizona/kfs/fp/document/DisbursementVoucherDocument.java

```diff
@@ -44,6 +44,9 @@
  * Document class override to ensure that the bank code is synchronized with the
  * payment method code.
  */
+// org.kuali.rice.kns.document.authorization.TransactionalDocumentPresentationController is deprecated
+// getReportable1099TransactionsFlag() and processAfterRetrieve() use unchecked Lists
+@SuppressWarnings({ "deprecation", "unchecked" })

```

Comment:
The TransactionalDocumentPresentationController is used in the prepareForSave method around line 89. Eclipse suggested both warnings be suppressed at the class level, which is why I made those notes, for any future developers that may touch this file.